lib/task/import/csvAuthorityRecordImportTask.class.php

```diff
@@ -535,29 +535,53 @@ public function execute($arguments = array(), $options = array())
               }
               else
               {
-                $relation = new QubitRelation;
-                $relation->subjectId = $sourceActorId;
-                $relation->objectId  = $targetActorId;
-                $relation->typeId    = $relationTypeId;
-
-                if ($self->rowStatusVars['date'])
-                {
-                  $relation->date = $self->rowStatusVars['date'];
-                }
-                if ($self->rowStatusVars['startDate'])
-                {
-                  $relation->startDate = $self->rowStatusVars['startDate'];
-                }
-                if ($self->rowStatusVars['endDate'])
+                // Check for pre-existing relationship before adding
+                $sql = "SELECT id FROM relation \r
+                  WHERE subject_id=:subject_id \r
+                  AND object_id=:object_id \r
+                  AND type_id = :type_id";
```

Comment:
Nitpick: Extra spaces around '=' compared to prev lines.

Reply:
Good spotting... I've added the spaces to all of them as it makes it a bit easier to read.